src/mimetypes.h

```diff
@@ -0,0 +1,141 @@
+inline constexpr std::string_view mimeAppMSWord{"application/msword"};
+inline constexpr std::string_view mimeAppVndXmlWord{"application/vnd.openxmlforMats-officedocument.wordprocessingml.document"};
+inline constexpr std::string_view mimeAppVndMSPpt{"application/vnd.ms-powerpoint"};
+inline constexpr std::string_view mimeVndOODoc{"application/vnd.oasis.openDocument.text"};
+inline constexpr std::string_view mimeAppZip{"application/zip"};
+inline constexpr std::string_view mimeAppWasm{"application/wasm"};
+inline constexpr std::string_view mimeAppOctetStream{"application/octet-stream"};
+
+inline constexpr std::string_view mimeAppFontTtf{"application/font-ttf"}; // TODO: in zimcreatorfs but not in table
+inline constexpr std::string_view mimeAppVndMsOpenType{"application/vnd.ms-opentype"}; // TODO: in zimcreatorfs but not in table
+
+inline constexpr std::string_view extHtml       { "html"};
+inline constexpr std::string_view extHtm        { "htm"};
+inline constexpr std::string_view extPng        { "png"};
+inline constexpr std::string_view extTiff       { "tiff"};
+inline constexpr std::string_view extTif        { "tif"};
+inline constexpr std::string_view extJpeg       { "jpeg"};
+inline constexpr std::string_view extJpg        { "jpg"};
+inline constexpr std::string_view extGif        { "gif"};
+inline constexpr std::string_view extSvg        { "svg"};
+inline constexpr std::string_view extTxt        { "txt"};
+inline constexpr std::string_view extXml        { "xml"};
+inline constexpr std::string_view extEpub       { "epub"};
+inline constexpr std::string_view extPdf        { "pdf"};
+inline constexpr std::string_view extOgg        { "ogg"};
+inline constexpr std::string_view extOgv        { "ogv"};
+inline constexpr std::string_view extJs         { "js"};
+inline constexpr std::string_view extJson       { "json"};
+inline constexpr std::string_view extCss        { "css"};
+inline constexpr std::string_view extOtf        { "otf"};
+inline constexpr std::string_view extSfnt       { "sfnt"};
+inline constexpr std::string_view extEot        { "eot"};
+inline constexpr std::string_view extTtf        { "ttf"};
+inline constexpr std::string_view extCollection { "collection"};
+inline constexpr std::string_view extWoff       { "woff"};
+inline constexpr std::string_view extWoff2      { "woff2"};
+inline constexpr std::string_view extVtt        { "vtt"};
+inline constexpr std::string_view extWebm       { "webm"};
+inline constexpr std::string_view extWebp       { "webp"};
+inline constexpr std::string_view extMp4        { "mp4"};
+inline constexpr std::string_view extDoc        { "doc"};
+inline constexpr std::string_view extDocx       { "docx"};
+inline constexpr std::string_view extPpt        { "ppt"};
+inline constexpr std::string_view extOdt        { "odt"};
+inline constexpr std::string_view extOdp        { "odp"};
+inline constexpr std::string_view extZip        { "zip"};
+inline constexpr std::string_view extWasm       { "wasm"};
```

Comment:
Why are these needed? They are not used anywhere outside `create_extMimeTypes()`.